db/model/model.js

```diff
@@ -152,3 +146,13 @@ exports.getArticlesByTopic = (topic = "PassBYChecker") => {
       .then(({ rows }) => rows);
   }
```

Comment:
your sql query needs to be refactored so that you still get comments as well as your topics if a user has a topic query. currently you are doing either or which is not what we want